psc-flink/src/main/java/com/pinterest/flink/streaming/connectors/psc/table/PscConnectorOptions.java

```diff
@@ -109,6 +109,16 @@ public class PscConnectorOptions {
 
     public static final ConfigOption<Integer> SINK_PARALLELISM = FactoryUtil.SINK_PARALLELISM;
 
+    public static final ConfigOption<Integer> SCAN_PARALLELISM =
```

Comment:
qq: this approach asks the user to supply the parallelism explicitly, right? Is there a way to infer (like what we do w/ batch sources like Iceberg tables), if user don't specify?